go.mod

```diff
@@ -1,34 +1,33 @@
 module github.com/kubernetes-csi/csi-lib-utils
 
-go 1.18
+go 1.19
```

Comment:
Please keep it at 1.18. 1.19 would indicate that the code in this repo needs 1.19, which isn't the case.